### PR TITLE
polish(pu): add config_spec_zh, basic_rl_zh, exploration_rl_zh, imitation_learning_zh

### PR DESCRIPTION
**Pull request by puyuan1996**
polish(pu): add first version of config_spec_zh, basic_rl_zh, exploration_rl_zh, imitation_learning_zh

### CODE REVIEW

File: source/02_algo/exploration_rl_zh.rst [hunk 1]
```diff
@@ -0,0 +1,248 @@
+强化学习中的探索机制
+====================
+
+本文主要参考这篇关于强化学习中探索策略的综述\ `博客 <https://lilianweng.github.io/posts/2020-06-07-exploration-drl/>`__\ 。
+
+问题与动机
+----------
+
+强化学习，是通过环境给出的奖励信号来指导策略的更新，力求获取最大累计折扣奖励，但在很多现实世界环境中奖励是稀疏的，甚至完全没有奖励，如何在这种情况下，引导智能体高效地探索状态和动作空间，进而找到最优策略完成任务？
```
Comment (lixl-st) [inline source/02_algo/exploration_rl_zh.rst:9]: 这句话不要全是"，"间隔，最好断开

File: source/02_algo/exploration_rl_zh.rst [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+   :math:`a` 的次数成反比的函数。
+
+-  Boltzmann Exploration：智能体从学习到的 Q 值对应的玻尔兹曼分布
+   (即对Q值 的logits执行softmax操作后得到的分布)
+   中采样动作，可以通过温度参数调整探索程度。
+
+当通过神经网络进行函数近似时，以下机制可用于在深度强化学习训练中得到更好的探索效果：
+
+-  Entropy
+   loss：通过在损失函数上增加额外的熵正则化项，来鼓励智能体执行更多样化的动作。
```
Comment (lixl-st) [inline source/02_algo/exploration_rl_zh.rst:56]: Loss 首字母大写

File: source/02_algo/exploration_rl_zh.rst [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+-  Epsilon-Greedy：在某一时刻，所有动作的选择概率都大于0。智能体以一个较小的概率\ :math:`\epsilon`\ 执行随机的动作（探索），以一个较大的概率\ :math:`1-\epsilon`\ 执行Q值最大的动作（利用）。
+
+-  Upper Confidence
+   Bounds：智能体贪婪地选取最大化置信上界\ :math:`\hat{Q}_{t}(a)+\hat{U}_{t}(a)`\ 的动作，其中\ :math:`\hat{Q}_{t}(a)`\ 是在
+   :math:`t` 时刻前与动作
+   :math:`a`\ 有关的平均奖励，\ :math:`\hat{U}_{t}(a)` 是一个与采取动作
+   :math:`a` 的次数成反比的函数。
+
+-  Boltzmann Exploration：智能体从学习到的 Q 值对应的玻尔兹曼分布
+   (即对Q值 的logits执行softmax操作后得到的分布)
```
Comment (lixl-st) [inline source/02_algo/exploration_rl_zh.rst:50]: 空格：的 logits 执行 softmax 操作

File: source/02_algo/exploration_rl_zh.rst [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+3. 基于存储的探索
+
+4. 直接探索
+
+5. 其他
+
+**经典的探索机制**
+------------------
+
+在传统的multi-armed bandit问题中，常用的经典探索机制包括：
```
Comment (lixl-st) [inline source/02_algo/exploration_rl_zh.rst:39]: 空格：在传统的 multi-armed bandit 问题中

File: source/02_algo/exploration_rl_zh.rst [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+
+**基于预测误差的内在奖励**
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+基于预测误差的内在奖励是\ **利用状态在某个预测问题 (通常是监督学习问题)
+上的预测误差来度量新颖性**\ 。根据在监督学习中神经网络拟合数据集的特性，如果智能体在某个状态\ :math:`s`\ 上预测误差越大，近似说明在状态\ :math:`s`
+附近的状态空间上智能体之前访问的次数少，从而该状态\ :math:`s`
+新颖性较大。
+
+预测问题往往是与环境的dynamics相关的问题，例如论文[3] `Curiosity-driven
```
Comment (lixl-st) [inline source/02_algo/exploration_rl_zh.rst:114]: 这个论文链接好像没弄好

File: source/02_algo/imitation_learning_zh.rst [hunk 6]
```diff
@@ -0,0 +1,127 @@
+Imitation learning
```
Comment (lixl-st) [inline source/02_algo/imitation_learning_zh.rst:1]: learning 首字母大写

File: source/20_spec/config_spec_zh.rst [hunk 7]
```diff
@@ -0,0 +1,155 @@
+DI-engine config规范
```
Comment (lixl-st) [inline source/20_spec/config_spec_zh.rst:1]: 格式：DI-engine Config 规范

File: source/02_algo/exploration_rl_zh.rst [hunk 8]
```diff
@@ -0,0 +1,248 @@
...
+
+-  环境给出的奖励很稀疏。智能体需要作出特定的序列动作才可能得到一个非零的奖励，如果每一步仅仅采用随机探索，很可能在整个学习过程中都遇不到一个非零的奖励。例如在
+   Montezuma's Revenge
+   里面智能体需要执行长序列动作，才能获取钥匙或者进入新的房间，这时才会有一个奖励。
+
+-  环境给出的奖励具有误导性。例如在 Pitfall
+   游戏里面，不仅奖励很稀疏，而且智能体的很多动作会得到一个负的奖励，智能体在学习到如何获取一个正的奖励之前，可能会由于这些负的奖励的存在，停在原地不动，导致缺乏探索。
+
+这时设计一个特别的探索机制就至关重要。
+因为如果智能体在有限步的训练之后变得贪婪，即在某个状态下只选择当前它认为最优的动作，它可能永远学不到最优策略，因为它很可能已经收敛到了次优策略，永远到达不了真正有意义的奖励大的状态。这即是所谓的探索与利用困境问题。通俗来讲，所谓探索：是指做你以前从来没有做过的事情，以期望获得更高的回报；所谓利用：是指做你当前知道的能产生最大回报的事情。
```
Comment (lixl-st) [inline source/02_algo/exploration_rl_zh.rst:22]: 探索和利用的定义放前面会不会好点

**Review by lixl-st — APPROVED** [review]
